utils/bazel/llvm-project-overlay/libc/test/src/__support/CPP/BUILD.bazel

```diff
@@ -0,0 +1,112 @@
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Tests for LLVM libc CPP functions.
+
+package(default_visibility = ["//visibility:public"])
```

Comment:
[nit] why ? these are tests...

Reply:
I've cargo culted it from the other tests `utils/bazel/llvm-project-overlay/libc/test/src/**/BUILD.bazel`.
I *think* this is because of our internal massaging script that tags everything as public on the "internal -> upstream" path. It's probably fixable but I don't want to bother for now :)